stata_kernel/docs/css/pandoc.css

```diff
@@ -0,0 +1,165 @@
+body {
+  font-family: 'Monospace';
+}
+
+blockquote {
+  /* color: #666666; */
+  margin: 0;
+  padding-left: 3em;
+  border-left: 0.5em solid /*#EEE*/;
+}
+
+hr {
+  display: block;
+  height: 2px;
+  border: 0;
+  border-top: 1px solid /* #aaa */;
+  border-bottom: 1px solid /* #eee */;
+  margin: 1em 0;
+  padding: 0;
+}
+
+p {
+  margin: 1em 0;
+}
+
+img {
+  max-width: 75%;
+}
+
+h1, h2, h3, h4, h5, h6 {
+  /* color: #111; */
+  line-height: 125%;
+  margin-top: 2em;
```

Comment:
Changing this to `0em` looks good in Hydrogen...
![image](https://user-images.githubusercontent.com/15164633/45131755-65555980-b15c-11e8-8d8a-dcf623342615.png)

Reply:
I don't see a difference in the Notebook

2em:
![image](https://user-images.githubusercontent.com/15164633/45131859-cbda7780-b15c-11e8-89e1-359ed86c1193.png)

0em:
![image](https://user-images.githubusercontent.com/15164633/45131874-e0b70b00-b15c-11e8-9597-db1a5b9b0ca4.png)

Reply:
Oh nevermind, I have to recompile each time...

Reply:
Even after recompiling it still looks fine with `0em` in the Notebook and better in Hydrogen:
![image](https://user-images.githubusercontent.com/15164633/45131976-2c69b480-b15d-11e8-96a5-345cc11e7d05.png)
![image](https://user-images.githubusercontent.com/15164633/45131984-312e6880-b15d-11e8-867c-46e1fea26d83.png)